src/udiskslinuxencrypted.c

```diff
@@ -408,8 +459,24 @@ handle_unlock (UDisksEncrypted        *encrypted,
   /* calculate the name to use */
   if (is_in_crypttab && crypttab_name != NULL)
     name = g_strdup (crypttab_name);
+  else {
+    if (is_luks)
+      name = g_strdup_printf ("luks-%s", udisks_block_get_id_uuid (block));
+    else
+      /* TCRYPT devices don't have a UUID, so we use the device number instead */
+      name = g_strdup_printf ("tcrypt-%" G_GUINT64_FORMAT, udisks_block_get_device_number (block));
+  }
+
+  /* save old encryption type to be able to restore it */
+  old_hint_encryption_type = udisks_encrypted_dup_hint_encryption_type(encrypted);
```

Comment:
Missing space after the function name.

Reply:
Fixed in 932aff4be696d43ad2a7b4117713443e93b23b05.